noisepagecontrol/control_plane/services/exploratory_worker_handler/exploratory_pg_status_types.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class ExploratoryPGStatusType(str, Enum):
+    PENDING = "PENDING"
+    READY = "READY"
+    STOPPED = "STOPPED"
```

Comment:
another state for shutting down?

Reply:
Considering /stop_exploratory_postgres is now synchronous call, do you think we still need SHUTTING_DOWN state?

Reply:
oh didnt know it was sync; cool